File: android/src/main/java/com/plugin/oss/imp/StsToken.java
package com.plugin.oss.imp;

public class StsToken {


    private String accessKeyId;
    private String accessKeySecret;
    private String securityToken;
    private String expiration;



    public String getAccessKeyId() {
        return accessKeyId;
    }

    public void setAccessKeyId(String accessKeyId) {
        this.accessKeyId = accessKeyId;
    }

    public String getAccessKeySecret() {
        return accessKeySecret;
    }

    public void setAccessKeySecret(String accessKeySecret) {
        this.accessKeySecret = accessKeySecret;
    }

    public String getSecurityToken() {
        return securityToken;
    }

    public void setSecurityToken(String securityToken) {
        this.securityToken = securityToken;
    }

    public String getExpiration() {
        return expiration;
    }

    public void setExpiration(String expiration) {
        this.expiration = expiration;
    }

    @Override
    public String toString() {
        return "StsToken{" +
                "accessKeyId='" + accessKeyId + '\'' +
                ", accessKeySecret='" + accessKeySecret + '\'' +
                ", securityToken='" + securityToken + '\'' +
                ", expiration='" + expiration + '\'' +
                '}';
    }
}

File: ios/Classes/imp/StsToken.swift
//
//  StsToken.swift
//  oss
//
//  Created by 胡杰 on 2019/5/29.
//

import UIKit

class StsToken: NSObject {

    var accessKeyId:String
    var accessKeySecret:String
    var securityToken:String
    var expiration:String
    
    init(accessKeyId:String,accessKeySecret:String,securityToken:String,expiration:String) {
        
        self.accessKeyId = accessKeyId
        self.accessKeySecret = accessKeySecret
        self.securityToken = securityToken
        self.expiration = expiration
        
    }
    
}
